HousingManagementSystemApi.Tests/HelpersTests/CosmosAddressQueryHelperTests.cs

```diff
@@ -0,0 +1,64 @@
+namespace HousingManagementSystemApi.Tests.HelperTests
+{
+    using HACT.Dtos;
+    using Helpers;
+    using Microsoft.Azure.Cosmos;
+    using Moq;
+    using Xunit;
+
+    public class CosmosAddressQueryHelperTests
+    {
+        private CosmosAddressQueryHelper systemUnderTest;
+        private Mock<Container> containerMock;
+        private Mock<FeedIterator<PropertyAddress>> feedIteratorMock;
+
+        public CosmosAddressQueryHelperTests()
+        {
+            this.containerMock = new Mock<Container>();
+            this.systemUnderTest = new CosmosAddressQueryHelper(this.containerMock.Object);
+            this.feedIteratorMock = new Mock<FeedIterator<PropertyAddress>>();
+        }
+
+        [Fact]
+        public void Test_Query_Definition_returned_with_querytext()
```

Comment:
The underscores in the test names are raised as warnings during the build due to breaking defined naming conventions - but in the case of test names it was decided to allow it.
Add a disable pragma to remove the warnings, i.e.
```suggestion
#pragma warning disable CA1707
        public void Test_Query_Definition_returned_with_querytext()
#pragma warning restore CA1707
```

The comment applies to each of the tests in this PR.